charts/vald/templates/gateway/meta/networkpolicy.yaml

```
@@ -0,0 +1,40 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+{{- $lb := .Values.gateway.lb -}}
+{{- $meta := .Values.gateway.meta -}}
+{{- if .Values.defaults.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ $meta.name }}-allow
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)